react/components/OrderInfo/PaymentMethod.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { FormattedPrice } from 'vtex.order-details'
+import { PaymentFlagPicker } from 'vtex.payment-flags'
+
+// Still needs to cover all posible values for `pay.group`
+const paymentGroupSwitch = (payment) => {
+  switch (payment) {
+    case 'creditCard':
+      return 'Cartão de crédito'
+    default:
+      break
+  }
+}
+
+const PaymentMethod = ({ payment }) => (
+  <div className="flex flex-column justify-around mr7-ns">
+    <p className="t-heading-6">{paymentGroupSwitch(payment.group)}</p>
+    <PaymentFlagPicker paymentSystem={payment.paymentSystem}>
+      {FlagComponent =>
+        FlagComponent && (
```

Comment:
What if the `FlagComponent` is not present?

Reply:
`FlagComponent` would not be present if `payment.paymentSystem` were a number that is not currently supported by `PaymentFlagPicker`. In that case, the component `PaymentFlagPicker` handles that and just doesn't render anything.